sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -22,19 +22,24 @@
 import collections
 import itertools
 import logging
-import Queue
+import queue
 import sys
 import threading
 import traceback
+from builtins import object
+from builtins import range
 from weakref import WeakValueDictionary
 
-import six
+from future import standard_library
+from future.utils import raise_
 
 from apache_beam.metrics.execution import MetricsContainer
 from apache_beam.runners.worker import statesampler
 from apache_beam.transforms import sideinputs
 from apache_beam.utils import counters
 
+standard_library.install_aliases()
```

Comment:
Do we want to call this before we do the import of "queue" above?  It looks like this only works because of an "accident" in that this module is imported after someone else already called `install_aliases()`.

Reply:
pylint errors on placing `install_aliases()` above the imports. That's the reasoning for this location.
I tested functionality of this placement and this order works as well:
test script (Python2):
Without `install_aliases()`:
```
>>> import sys
>>> sys.intern
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'intern'
```
With `install_aliases()` after import sys
```
>>> import sys
>>> from future import standard_library
>>> standard_library.install_aliases()
>>> sys.intern
<built-in function intern>
```